Tests/Tests/EventMatcherSpec.swift

```diff
@@ -85,6 +89,17 @@ class EventMatcherSpec: QuickSpec {
                     }.to(throwError(EventMatcherError.couldntFindRequestedSetOfEvents))
                 }
 
+                it("will throw error if the event to remove isn't persistent or the event weren't found inside persistent events") {
```

Comment:
1. "... the event wasn't"
2. "or" suggests this test should be split into 2 tests to cover all scenarios
3. CustomEvent can't be a presistent event. I don't understand how persistent events are used in this test case 🤔

Reply:
1. Yes, I change this.
2. I change the description to only one because I think the conditional is to check whether the event is a persistent event or not.
3. That's why we use this case to cover line 106 in EventMatcher to return an `error:EventMatcherError.couldntFindRequestedSetOfEvents` because the CustomEvent is not a persistent one.

Reply:
Ok, I see. The name is still a bit confusing because the EventMatcher is able to remove an event that isn't persistent without throwing an error. Actually, please move content of this test to "will throw error if events for given campaign weren't found" test and replace its content. The replaced content should be put under a new test with name "will throw error if persistent event for given campaign wasn't found"